src/rx-query.ts

```diff
@@ -555,7 +555,13 @@ export class RxQueryBase<
         }
         // eslint-disable-next-line no-console
         console.time(`Restoring persistent querycache ${this.toString()}`);
-        const lwt = (await this._persistentQueryCacheBackend.getItem(`qc:${persistentQueryId}:lwt`)) as string | null;
+
+        const results = await Promise.all([
+            this._persistentQueryCacheBackend.getItem(`qc:${persistentQueryId}:lwt`),
+            this._persistentQueryCacheBackend.getItem(`qc:${persistentQueryId}:lb`),
+        ]);
+        const [lwt, limitBufferIds] = [results[0] as string | null, results[1] as string[] | null];
```

Comment:
you can destructure this

Reply:
you can cast the promise result to `(string[] | null)[]`

Reply:
done, indirectly